tasks/insights-client.yml

```diff
@@ -36,6 +36,27 @@
     insertafter: "#auto_update"
     line: auto_update={{ rhc_insights.autoupdate | d(true) | bool }}
 
+- name: Configure ansible host
+  when: rhc_insights.ansible_host | d() != None
```

Comment:
ah, this check needs to be tweaked:
```yaml
  when:
    - not rhc_insights.ansible_host is none
    - rhc_insights.ansible_host != omit
```
Since `rhc_insights.ansible_host` is defined in `defaults/main.yml`, then it will be always defined.

The explicit check for `omit` is needed to avoid using the special value `omit` as real value; `omit` would be passed as role parameter in case there is no actual value for it -- e.g.:
```yaml
  rhc_insights:
    ansible_host: "{{ some_string if enable_host and some_string is defined else omit }}"
```